app/views/region_admin/recipients/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<h2>Recipients</h2>
+
+<div style="text-align: right;">
+  <%= form_tag("/locations/new", :method => "get") do %>
+    New Recipient Location For <%= select_tag(:region_id, options_for_select(@regions.collect{ |r| [r.name,r.id] })) %>
+    <%= submit_tag("Go") %>
+  <% end %>
+</div>
+
+<% if @locations.empty? %>
+  <p>No locations.</p>
+<% else %>
+  <table id="data_table" class="table table-striped">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Address</th>
+        <th>Link</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @locations.each do |location| %>
+        <tr>
+          <td>
+            <%= link_to location.name.presence || "Unnamed location", location_url(location) %> <%= link_to "(website)", location.website if location.website.present? %>
+          </td>
+          <td>
+            <%= location.address.gsub("\n","<br>").html_safe unless location.address.nil? %>
+          </td>
+          <td>
+            <% if location.receipt_key.present? %>
+              <%= link_to 'Hud', "/locations/hud?id=#{location.id}&key=#{location.receipt_key}" %><br>
+            <% end %>
+            <%= link_to "Edit", edit_location_url(location) %><br>
+            <%= link_to "Delete", location_url(location), :confirm => "Are you sure?", :method => :delete %>
+          </td>
```

Comment:
It looks like this view is missing a few table columns that were originally shown when this list was rendered with `locations/index.html.erb`. Was this intentional?

https://github.com/boulder-food-rescue/food-rescue-robot/blob/rf/recipients-controller-spec/app/views/locations/index.html.erb#L40-L49